### ISSUE TITLE
MotB: Rest impossible with package installed

### ISSUE THREAD

**Issue by Scrapulous** [nwn2fixes/player#40]
Clicking "Rest" just closes the rest window without any effect.

**Comment by kevL** [comment]
I can't replicate it. I loaded an old MotB save with a clean installation folder and an override folder with only Nwn2Fixes in it. No custom guis etc etc.

rest went fine here ...

i'll ask on the forum thread to see if anyone else has an issue with MotB resting

**Comment by kevL** [comment]
https://forum.neverwintervault.org/t/nwn2fixes/595/368?u=kevl_s

**Comment by jigsaw717** [comment]
I just started MOTB and im having this issue, i narrowed it down to something in the 'base' folder preventing me from resting, when i remove the entire folder, it works fine.

**Comment by kevL** [comment]
so it's still there huh?

It seems like half the people using nwn2fixes get the problem and the other half don't ...


will take a look at those files, and probly take them out,

**Comment by kevL** [comment]
@jigsaw717 Are you using the latest fixes?

2020_mar_10.ver

**Comment by jigsaw717** [comment]
i am indeed

**Comment by kevL** [comment]
ok. Am going to try a different approach ...

the underlying bug is that, in SoZ on the overlandmap, the player can choose a dialog option to Intimidate an encounter before going into battle but EffectShaken() and EffectOffGuard() use negative values when constructing the effects. Positive values should be used in 'ginc_overland' instead (the constructors for the effects will subtract the positive values).

Tracing those functions back through several layers, they could be called by 'gui_rest'. HOWEVER the only place i *know* of them being used is on the SoZ OL. And those are handled by just two scripts: 'ga_initiate_encounter' and 'ga_mated_encounter'.

So basically i'm going to delete gui_rest.ncs and gui_rest.NSS and x2_inc_restsys.nss and ginc_restsys.NSS. But keep 'ginc_overland' with the core fix, as well as 'ga_initiate_encounter' and 'ga_mated_encounter'.

and see how that plays,

**Comment by jigsaw717** [comment]
I've got no idea what any of that means but i hope it works out well :D

**Comment by kevL** [comment]
we'll see :)

https://github.com/nwn2fixes/player/commit/f8424f94f70ef35d524fa1e1b6b2feee20ab94b5

The notes are just to get me straight on what's happening, and for anyone who reads it and wants to poke away at it themselves
